pom.xml

```diff
@@ -98,13 +100,13 @@
     <dependency>
       <groupId>xml-apis</groupId>
       <artifactId>xml-apis</artifactId>
-      <version>1.4.01</version><!--$NO-MVN-MAN-VER$ -->
+      <version>2.0.2</version>      <!--$NO-MVN-MAN-VER$ -->
     </dependency>
 
     <dependency>
       <groupId>com.github.jsonld-java.jsonld-java</groupId>
       <artifactId>jsonld-java</artifactId>
-      <version>0.12.5</version>
+      <version>0.13.4</version>
```

Comment:
An exclusion for jackson-databind is necessary due to a vulnerability in it:
- [CVE-2020-36518](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-36518)

We should explicitly include a version like 2.13.3.

Reply:
This can be solved by switching to the new Spring-boot 2.7.0 (or greater).